keyservice/server.go

```diff
@@ -11,7 +13,10 @@ import (
 )
 
 // Server is a key service server that uses SOPS MasterKeys to fulfill requests
-type Server struct{}
+type Server struct {
+	// Prompt indicates whether the server should prompt before decrypting or encrypting data
+	Prompt bool
```

Comment:
Perhaps worthy of a separate PR, but having an option to log requests (same logical flow as prompt, but simply print the incoming request instead of prompting) could see use.